README.md

```diff
@@ -22,7 +22,11 @@ Please notice that the extension uses `.vscode/.react` directory at the project
 
 ## Debugging React Native applications
 
-Click the debug icon ![Choose React Native debugger](images/debug-view-icon.png) in the View bar, and then click the configuration (gear) icon ![Configure-gear](images/configure-gear-icon.png), then choose the React Native debug environment.
+To start debugging you need to create debug configurations in `.vscode/launch.json` first. You can always do it by opening it and clicking at `Add Configuration...` button and then choosing needed debug configuration.
+
+![Choose React Native debug configuration](images/add-debug-configuration.png)
```

Comment:
Can we capture a gif and include that list we do everywhere else?  Its better than having Red arrows :)

Reply:
@dhanvikapila Sure, I'll take care of it.